R/bootstrap.nlsfit.R

```diff
@@ -1,100 +1,47 @@
 bootstrap.nlsfit <- function(fn,
                              gr = NULL,
+                             dfn = NULL,
```

Comment:
what is `dfn` in contrast to `gr`?

Reply:
gr = d(fn) / d(par)
This is the gradient in parameter space.

dfn = d(fn) / dx
This is the derivative by the canonical variable. Only needed for the gradient function with xyerrors.